src/upload/admin/controller/extension/payment/webpay_rest.php

```diff
@@ -182,25 +135,22 @@ public function index() {
         $args = array(
             'MODO' => $this->default_config['test_mode'],
             'COMMERCE_CODE' => $this->default_config['commerce_code'],
-            'PRIVATE_KEY' => $this->default_config['private_key'],
-            'PUBLIC_CERT' => $this->default_config['public_cert'],
+            'API_KEY' => $this->default_config['api_key'],
             'ECOMMERCE' => 'opencart'
         );
 
-        if (isset($this->request->post['payment_webpay_commerce_code'])) {
+        if (isset($this->request->post['payment_webpay_rest_commerce_code'])) {
             $args = array(
-                'MODO' => $this->request->post['payment_webpay_test_mode'],
-                'COMMERCE_CODE' => $this->request->post['payment_webpay_commerce_code'],
-                'PRIVATE_KEY' => $this->request->post['payment_webpay_private_key'],
-                'PUBLIC_CERT' => $this->request->post['payment_webpay_public_cert'],
+                'MODO' => $this->request->post['payment_webpay_rest_test_mode'],
+                'COMMERCE_CODE' => $this->request->post['payment_webpay_rest_commerce_code'],
+                'API_KEY' => $this->request->post['payment_webpay_rest_api_key'],
                 'ECOMMERCE' => 'opencart'
             );
-        } else if ($this->config->get('payment_webpay_commerce_code')) {
+        } else if ($this->config->get('payment_webpay_rest_commerce_code')) {
```

Comment:
This is just for the example code, right? otherwise, why would you rename this?

Reply:
It it because `payment_webpay_commerce_code` was used by the old plugin, and eventually someone can install both plugins at the same time. This change allows both plugins to work simultaneously.

Reply:
My 2 cents: I'd handle the two-plugins-at-the-same-time as 

- A robustness case (i.e, we don't break anybody's site because he forgot to uninstall the old plugin or installed the new plugin without setting it up yet) 

- A transition-path case (i.e, if I had a store I wouldn't completely nuke the old setup before trying the new one, so it is totally expected that people will have both plugins installed and even active at the same time before turning off the old one). 

If you agree with that, please remember both cases for all of the plugins :) cc: @gonzunigad @a1b4

Reply:
Great. It would be great if someone can approve this PR then :)
For other plugins I also expect the two-plugins-at-the-same-time feature. We can check that case by case.

Reply:
I think you need @a1b4's help on how to call the team's attention here. Or @eugenioLopezRamos's who has the blocking review

Reply:
I agree to change names to allow that the users install the both plugins, but first we need to create the Issue in the plugins that does not have this. So I will wait by those Issues :)